Makefile

```diff
@@ -17,11 +17,16 @@ endif
 
 -include $(OUT)/local.mk
 
-ifeq ("$(BUILD_DEBUG)","1")
-CFLAGS += -Og -g3 -DENABLE_DEBUG
+ifneq ("$(BUILD_DEBUG)","0")
+    CFLAGS += -Og -g3 -DENABLE_DEBUG
+    ifneq ("$(BUILD_DEBUG)","1")
+        # Sanitizer checking
```

Comment:
We need extra `mk/sanitizers.mk` to filter out invalid input strings.

Reply:
Also, gcc and/or clang versions should be validated in order to enable sanitizers.

Reply:
Since not every Sanitizer is supported on `arm-linux`, I would focus on `x86` platform only.

Reply:
Done in the new file `mk/sanitizers.mk`.